libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -521,6 +521,7 @@ class SRV_Channels {
 
     // disable output to a set of channels given by a mask. This is used by the AP_BLHeli code
```

Comment:
this existing comment should perhaps be, "// disable **PWM** output to a set of channels..."

Reply:
fixed